public/index.html

```diff
@@ -32,6 +32,8 @@
     <link rel="apple-touch-icon" sizes="144x144" href="img/apple-icon-144x144.png" />
     <link rel="stylesheet" type="text/css" href="style.css">
     <link rel="stylesheet" type="text/css" href="css/st-tailwind.css">
+    <link rel="stylesheet" type="text/css" href="css/animations.css">
```

Comment:
You don't have to import all of the CSS files in the HTML, just add `@import` in style.css

Reply:
Done. I think I just missed that, because we have lots of other CSS files listed there. Maybe we could/should move them too?

(Thank god I implemented recursive import checking in my dynamic focus styles, lol)